src/main/java/omero/gateway/model/PlateData.java

```diff
@@ -135,6 +137,13 @@ public PlateData(Plate plate)
 
     // IMMUTABLES
 
+    public Set<WellData> getWells() {
+        Set<WellData> wells = new HashSet();
+        for (Well w : asPlate().copyWells())
+            wells.add(new WellData(w));
+        return wells;
```

Comment:
other methods returning list return ``null`` for empty collections. This does not match since an empty list will be returned if the plate has no wells

Reply:
Indeed, thanks 👍